pdns/pdns_recursor.cc

```diff
@@ -637,6 +637,20 @@ static void protobufLogResponse(const std::shared_ptr<RemoteLogger>& logger, con
 }
 #endif
 
+void handleRPZCustom(const DNSRecord& spoofed, const QType& qtype, SyncRes& sr, int& res, vector<DNSRecord>& ret)
+{
+  if (spoofed.d_type == QType::CNAME) {
+    sr.d_wantsRPZ = false;
```

Comment:
I wonder if we should save the existing value of `d_wantsRPZ` to restore it afterwards? I doesn't seem to matter right now, but it might prevent surprises in the future.